makefiles/eclipse.inc.mk

```diff
@@ -4,6 +4,8 @@
 eclipsesym: $(CURDIR)/eclipsesym.xml
 eclipsesym.xml: eclipsesym
 
+RIOT_CONFIG_MODULE_SHELL=y
```

Comment:
Could it be this line maybe :grin:? This will be removed. It just here to check the `buildsystem` check extension. And of course this does not work without `TEST_KCONFIG!=1` for the reasons you gave ;-)